test/reviewRenderer/helpers/pci/test.js

```diff
@@ -0,0 +1,11 @@
+define(['jquery', 'taoQtiItem/reviewRenderer/helpers/pci'], function ($, pci) {
+    'use strict';
+
+    QUnit.test('isInteractionDisabled', function (assert) {
+        const isInteractionDisabledForTextReader = pci.isInteractionDisabled('textReaderInteraction');
+        const isInteractionDisabledForAudioPci = pci.isInteractionDisabled('"audioRecordingInteraction"');
```

Comment:
Should it be both quotes and double quotes here?

Reply:
removed